src/components/mission-planning/ContextMenu.vue

```diff
@@ -110,6 +110,18 @@
           ></v-icon>
           <span class="text-white text-sm ml-4">{{ pathCreationButtonText }}</span>
         </v-list-item>
+        <v-divider />
+        <v-list-item class="flex items-center gap-x-2 pb-2" @click="handleSetHomePosition">
```

Comment:
Ideally this menu would be radial:
![image](https://github.com/user-attachments/assets/0faa01ee-47a9-4bce-9245-69acf8d5618e)

but fair enough if that's not included in this PR.